src/api/Methods.ts

```diff
@@ -178,6 +184,30 @@ export function findInstances(app: AppIdentifier): Promise<AppIdentifier[]> {
   return rejectIfNoGlobal(() => window.fdc3.findInstances(app));
 }
 
+/**
+ * Check if the given context is a standard context type.
+ * @param contextType
+ */
+export function isStandardContextType(contextType: ContextType): contextType is StandardContextType {
+  return StandardContextsSet.has(contextType as StandardContextType);
+}
+
+/**
+ * Check if the given intent is a standard intent.
+ * @param intent
+ */
+export function isStandardIntent(intent: Intent): intent is StandardIntent {
+  return StandardIntentsSet.has(intent as StandardIntent);
+}
+
+/**
+ * Get the possible context types for a given intent.
+ * @param intent
+ */
+export function getPossibleContextsForIntent<I extends StandardIntent>(intent: I): ContextTypeFor<I>[] {
```

Comment:
@kriswest  Correct me if I'm wrong, but discussion on the call was in favor of the isStandard* being fine but getPossibleContextsForIntent be deferred for future/separate discussion?

Reply:
Correct - the 'possible contexts' content in the intent specifications was intended to be informative but isn't restrictive (I'm told), particularly as proprietary contexts are allowed. The possible contexts don't exist in code or schema anywhere (until this PR).

Apologies, I haven't had time to circle around to fully review this issue and the discussion yet, but I have posted the minutes from the meeting at: 
- #1160